framework/scenario/tests/token_properties_result_test.rs

```diff
@@ -0,0 +1,48 @@
+use multiversx_sc::{codec::TopDecodeMulti, types::TokenPropertiesResult};
+
+fn hex_decode(hex: &[u8]) -> Option<Vec<u8>> {
```

Comment:
We already have a call data decode facility, it's called HexCallDataDeserializer. Maybe it's worth picking it up and dusting it off.